lib/puppet/type/filebucket.rb

```diff
@@ -42,19 +42,13 @@ module Puppet
     end
 
     newparam(:server) do
-      desc "The server providing the remote filebucket service. Defaults to the
-        value of the `server` setting (that is, the currently configured
-        puppet master server).
+      desc "The server providing the remote filebucket service.
```

Comment:
I wasn't sure if I should add any indication of the logic the indirector goes through to return a server to use